recipes/yasm/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, MSBuild
+import os
+
+
+class YASMInstallerConan(ConanFile):
+    name = "yasm"
+    version = "1.3.0"
+    url = "https://github.com/bincrafters/conan-yasm_installer"
+    description = "Yasm is a complete rewrite of the NASM assembler under the “new” BSD License"
+    license = "BSD"
+    exports_sources = ["LICENSE"]
```

Comment:
```suggestion
```

Reply:
maybe we can add automated checks into hook for this and others?
it's pretty boring to do all these adjustments after PR every time.
my process is:
- copy recipe from bincrafters
- add `conandata.yml` and `config.yml`
- fix any errors conan center hook reported
as a contributor, after that, I would expect an approval, as soon as CI passes.

Reply:
We are still polishing the conan center hook and this first PRs help us with the task. If you encounter repetitive things like this that are not blocked by the Hook, please open an issue in the repo to improve it and help with the onboarding of future people to conan center index. Thanks!